src/components/projectRegister/ValidationModal.tsx

```
@@ -0,0 +1,293 @@
+  const step2Items = validationItems.filter((item) => item.step === 2)
+  const step3Items = validationItems.filter((item) => item.step === 3)
+  const step1OptionalItems = optionalItems.filter((item) => item.step === 1)
+
+  return (
+    <div className="fixed inset-0 z-50 flex items-center justify-center">
+      {/* 오버레이 */}
+      <div className="fixed inset-0 bg-black/60" onClick={onClose} />
+
+      {/* 모달 */}
+      <div className="relative z-10 w-full max-w-lg rounded-3xl bg-white p-6 shadow-lg max-h-[80vh] overflow-y-auto">
+        <div className="flex items-center justify-between mb-6">
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)